src/main/java/com/ylysenkova/movieland/dao/jdbc/JdbcCountryDao.java

```diff
@@ -36,6 +36,9 @@
     private String getCountryByMovieId;
     @Autowired
     private String getAllCountries;
+    @Autowired
+    private String removeLinkCountryMovie;
+
```

Comment:
Would be better add SQL to names of Strings which contain SQL statements. (Fix is not mandatory)